crates/spicepod/src/semantic.rs

```diff
@@ -152,6 +152,8 @@ pub struct FullTextSearchConfig {
         skip_serializing_if = "Option::is_none"
     )]
     pub row_ids: Option<Vec<String>>,
+
+    pub mode: Mode,
```

Comment:
We'll need to get this UX reviewed - I would also think we'd probably want a way to control where the FTS index lives on disk.